src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Link, List, ListItem } from "@mui/joy";
+import NextLink from "next/link";
+
+export function NavBar() {
+    return (
+        <header>
+            <nav className="navigation">
+                <List
+                    orientation="horizontal"
+                    sx={{
+                        bgcolor: "background.body",
+                        borderRadius: "4px",
+                        maxWidth: "fit-content",
+                        "--ListItem-radius": "8px",
+                    }}
+                >
+                    <ListItem>
+                        <NextLink href="/dashboard" legacyBehavior passHref>
```

Comment:
Have you tried using the  `component` prop for the `Link` component to avoid using `legacyBehaviour` for NextLink?

Reply:
yes